app/helpers/application_helper.rb

```diff
@@ -32,8 +32,7 @@ def user_works_page?
 
   def works_page?
     params[:controller] == 'works' &&
-    (params[:action] == 'on_air' ||
-     params[:action] == 'season' ||
+    (params[:action] == 'season' ||
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.